lib/ravelin/event.rb

```diff
@@ -63,6 +64,8 @@ def validate_top_level_payload_params
       when :checkout
         validate_payload_inclusion_of :customer, :order,
           :payment_method, :transaction
+      when :reclaim
+        validate_payload_inclusion_of :customers, :source
```

Comment:
Isn't the source always ATO?

Reply:
It is, though I'm struggling as to how to expose it when it's defined as a constant. We'll just set it in our client instead - that may be easier...